antispam.py

```diff
@@ -0,0 +1,55 @@
+import time
+import logging
+from util import singleton
+
+logger = logging.getLogger(__name__)
+
+
+@singleton
+class AntiSpam:
+
+    def __init__(self, count=5, timeout=10) -> None:
+        """
+        Cleat instance of the class
+        :param count: of actions to be allowed within the configured time
+        :param timeout: for which the actions are remembered and counted
+        """
+        self._lookup = {}
+        self._count = count
+        self._timeout = timeout
+
+    def is_spam(self, identifier):
+        """
+        check if the current identifier has exceeded the amount of invocations in the given threshold
+        :param identifier: string: unique identifier for the action to check against, e.g. user_id, user_id+operation
+        :return: true if this is not spam, false otherwise
```

Comment:
either method name or comment is wrong. based on function code the comment should be wrong